### ISSUE TITLE
The fedora:20 image has unpushed packages from updates-testing

### ISSUE THREAD

**Issue by adelton** [fedora-cloud/docker-brew-fedora#19]
This is a copy of https://bugzilla.redhat.com/show_bug.cgi?id=1248467:

Description of problem:

It seems fedora:20 image has openssl-libs from updates-testing, which is not enabled, preventing installation of openssl.

Version-Release number of selected component (if applicable):

fedora:20 image 1b48ab88a33e.

How reproducible:

Deterministic.

Steps to Reproduce:
1. # docker run -ti fedora:20 yum install openssl

Actual results:

Resolving Dependencies
--> Running transaction check
---> Package openssl.x86_64 1:1.0.1e-42.fc20 will be installed
--> Processing Dependency: openssl-libs(x86-64) = 1:1.0.1e-42.fc20 for package: 1:openssl-1.0.1e-42.fc20.x86_64
--> Processing Dependency: make for package: 1:openssl-1.0.1e-42.fc20.x86_64
--> Running transaction check
---> Package make.x86_64 1:3.82-19.fc20 will be installed
---> Package openssl.x86_64 1:1.0.1e-42.fc20 will be installed
--> Processing Dependency: openssl-libs(x86-64) = 1:1.0.1e-42.fc20 for package: 1:openssl-1.0.1e-42.fc20.x86_64
--> Finished Dependency Resolution
Error: Package: 1:openssl-1.0.1e-42.fc20.x86_64 (updates)
           Requires: openssl-libs(x86-64) = 1:1.0.1e-42.fc20
           Installed: 1:openssl-libs-1.0.1e-45.fc20.x86_64 (@fedora-updates-testing/$releasever)
               openssl-libs(x86-64) = 1:1.0.1e-45.fc20
           Available: 1:openssl-libs-1.0.1e-30.fc20.x86_64 (fedora)
               openssl-libs(x86-64) = 1:1.0.1e-30.fc20
           Available: 1:openssl-libs-1.0.1e-42.fc20.x86_64 (updates)
               openssl-libs(x86-64) = 1:1.0.1e-42.fc20
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest

Expected results:

No error, installation of openssl passes.

Additional info:

While Fedora 20 EOL'ed on 2015-06-23, it seems something has rebuilt the image on or after that date because http://koji.fedoraproject.org/koji/buildinfo?buildID=664541 as built the same day.

It'd be nice to have reasonably working Fedora 20 image around, we use it for upgrade testing.

**Comment by adelton** [comment]
I understand Fedora 20 is EOL'ed but we use it as basis of testing FreeIPA upgrades and besides, if the image is in the index, it should have easily usable content.

**Comment by MartinMikita** [comment]
+1 We need it to build our application, which requires `openssl-devel`. We are having customers with Fedora 20, that are not able to update. Why is damaged docker image pushed into public? How could we get older working docker image for `fedora:20`?

**Comment by adelton** [comment]
Filed as https://fedorahosted.org/spin-kickstarts/ticket/55 per https://github.com/fedora-cloud/docker-brew-fedora/commit/b00e7a78976ea610c21c5ffb999a39b3040e3d5f.

**Comment by petems** [comment]
For anyone out there who hits this:

```
yum distro-sync
yum install openssl-dev
```

Fixed it for me on an image forked from Fedora 20 👍
